packages/store/src/space.ts

```diff
@@ -27,6 +27,8 @@ export class Space<
    */
   protected readonly _ySpace: Y.Map<State[keyof State]>;
 
+  shouldTransact = true;
```

Comment:
Should be `private _shouldTransact` 😝

Reply:
Okay, move _shouldTransact to Page class now~